<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionForm</class>
 <widget class="QWidget" name="TransactionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="flashMsgUp">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>280</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QPushButton" name="pushButtonDelete">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>220</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="dateEditPaymentDate">
          <property name="geometry">
           <rect>
            <x>204</x>
            <y>50</y>
            <width>101</width>
            <height>24</height>
           </rect>
          </property>
          <property name="displayFormat">
           <string>dd/MM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonSave">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>220</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>50</y>
            <width>130</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Payment Date</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditPaidAmount">
          <property name="geometry">
           <rect>
            <x>204</x>
            <y>110</y>
            <width>141</width>
            <height>23</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonCancel">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>220</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBoxPayMode">
          <property name="geometry">
           <rect>
            <x>204</x>
            <y>170</y>
            <width>141</width>
            <height>23</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>112</y>
            <width>130</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Paid Amount</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>170</y>
            <width>130</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Pay Mode</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>141</y>
            <width>130</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Paid By</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditPaidBy">
          <property name="geometry">
           <rect>
            <x>204</x>
            <y>140</y>
            <width>141</width>
            <height>23</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>80</y>
            <width>130</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Invoice No</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditInvoiceNo">
          <property name="geometry">
           <rect>
            <x>204</x>
            <y>80</y>
            <width>141</width>
            <height>23</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dateEditPaymentDate</tabstop>
  <tabstop>lineEditInvoiceNo</tabstop>
  <tabstop>lineEditPaidAmount</tabstop>
  <tabstop>lineEditPaidBy</tabstop>
  <tabstop>comboBoxPayMode</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
